pkg/http/api.go

```diff
@@ -437,7 +437,7 @@ func (a *api) constructConfigurationEndpoints() []Endpoint {
 		},
 		{
 			Methods: []string{fasthttp.MethodPost},
-			Route:   "configuration/{configurationSubscribeID}/unsubscribe",
+			Route:   "configuration/{storeName}/{configurationSubscribeID}/unsubscribe",
```

Comment:
Could we have a test for this?

Reply:
I don't see any existing tests for the "https://v1-8.docs.dapr.io/reference/api/configuration_api/"

Reply:
Please add it.

Reply:
func TestV1Alpha1ConfigurationUnsubscribe(t *testing.T) {
	fakeServer := newFakeHTTPServer()

	var fakeConfigurationStore configuration.Store = &fakeConfigurationStore{}

	storeName := "store1"
	badStoreName := "nonExistStore"

	fakeConfigurationStores := map[string]configuration.Store{
		storeName: fakeConfigurationStore,
	}
	testAPI := &api{
		resiliency:          resiliency.New(nil),
		configurationStores: fakeConfigurationStores,
	}
	fakeServer.StartServer(testAPI.constructConfigurationEndpoints())

	t.Run("subscribe and unsubscribe configurations", func(t *testing.T) {
		
		apiPath1 := fmt.Sprintf("v1.0-alpha1/configuration/%s/subscribe", storeName)
		resp1 := fakeServer.DoRequest("GET", apiPath1, nil, nil)
		assert.Equal(t, 200, resp1.StatusCode, "subscribe configuration store, should return 200")


		rspMap1 := resp1.JSONBody.(interface{})
		assert.NotNil(t, rspMap1)
		//rspMap1.(map[string]interface{})["id"].(string)
		
		apiPath2 := fmt.Sprintf("v1.0-alpha1/configuration/%s/%s/unsubscribe", storeName,rspMap1.(map[string]interface{})["id"].(string))
		
		resp2 := fakeServer.DoRequest("POST", apiPath2, nil, nil)

		assert.Equal(t, 204, resp2.StatusCode, "unsubscribe configuration store,should return 204")
	})
}

Reply:
```suggestion
  like this at /pkg/http/api_test.go
  func TestV1Alpha1ConfigurationUnsubscribe(t *testing.T) {
	fakeServer := newFakeHTTPServer()

	var fakeConfigurationStore configuration.Store = &fakeConfigurationStore{}

	storeName := "store1"

	fakeConfigurationStores := map[string]configuration.Store{
		storeName: fakeConfigurationStore,
	}
	testAPI := &api{
		resiliency:          resiliency.New(nil),
		configurationStores: fakeConfigurationStores,
	}
	fakeServer.StartServer(testAPI.constructConfigurationEndpoints())

	t.Run("subscribe and unsubscribe configurations", func(t *testing.T) {
		
		apiPath1 := fmt.Sprintf("v1.0-alpha1/configuration/%s/subscribe", storeName)
		resp1 := fakeServer.DoRequest("GET", apiPath1, nil, nil)
		assert.Equal(t, 200, resp1.StatusCode, "subscribe configuration store, should return 200")


		rspMap1 := resp1.JSONBody.(interface{})
		assert.NotNil(t, rspMap1)
		//rspMap1.(map[string]interface{})["id"].(string)
		
		apiPath2 := fmt.Sprintf("v1.0-alpha1/configuration/%s/%s/unsubscribe", storeName,rspMap1.(map[string]interface{})["id"].(string))
		
		resp2 := fakeServer.DoRequest("POST", apiPath2, nil, nil)

		assert.Equal(t, 204, resp2.StatusCode, "unsubscribe configuration store,should return 204")
	})
}
```

Reply:
https://v1-8.docs.dapr.io/reference/api/configuration_api/#unsubscribe-configuration describes,unsubscribe action is GET, but here is POST, is this normal?
`Methods: []string{fasthttp.MethodPost},
			Route:   "configuration/{configurationSubscribeID}/unsubscribe",`

Reply:
No, this should be `GET`. Good catch! Please change the verb.

Reply:
> 不，这应该是 。抓得好！请更改动词。`GET`

 done